### ISSUE TITLE
Build failure for packmol on jenkins osx

### ISSUE THREAD

**Issue by jchodera** [omnia-md/conda-recipes#73]
The build for `packmol` fails because `build.sh` tries to use `/usr/bin/gfortran`; note that this is often installed in a different location on `osx` (e.g. `/usr/local/bin/gfortran`):
```
BUILD START: packmol-1.0.0-0
Fetching package metadata: ....
Solving package specifications: .+ export 'CFLAGS=-I/Users/jenkins-slave/workspace/conda-omnia-release-osx-2/miniconda/envs/_build/include -arch x86_64'
+ CFLAGS='-I/Users/jenkins-slave/workspace/conda-omnia-release-osx-2/miniconda/envs/_build/include -arch x86_64'
+ export 'LDFLAGS=-L/Users/jenkins-slave/workspace/conda-omnia-release-osx-2/miniconda/envs/_build/lib -arch x86_64'
+ LDFLAGS='-L/Users/jenkins-slave/workspace/conda-omnia-release-osx-2/miniconda/envs/_build/lib -arch x86_64'
+ make
 ------------------------------------------------------ 
 Compiling packmol with /usr/bin/gfortran 
 Flags: -O3 -ffast-math  
 ------------------------------------------------------ 
make: /usr/bin/gfortran: Permission denied
make: *** [serial] Error 1

The following NEW packages will be INSTALLED:

    zlib: 1.2.7-1

Linking packages ...
 |                                                                       |   0%
[zlib-1.2.7-1 /Users/jenkins-slave/workspace/conda-omnia-release] |######| 100%
[      COMPLETE      ] |#################################################| 100%
Removing old work directory
Source cache directory is: /Users/jenkins-slave/workspace/conda-omnia-release-osx-2/miniconda/conda-bld/src_cache
Downloading source to cache: packmol.tar.gz
packmol.tar.gz   0% |                              | ETA:  --:--:--   0.00  B/s
packmol.tar.gz  15% |####                           | ETA:  0:00:00  16.93 MB/s
packmol.tar.gz  31% |#########                      | ETA:  0:00:00  24.98 MB/s
packmol.tar.gz  47% |##############                 | ETA:  0:00:00  29.71 MB/s
packmol.tar.gz  63% |###################            | ETA:  0:00:00  32.96 MB/s
packmol.tar.gz  79% |########################       | ETA:  0:00:00  35.37 MB/s
packmol.tar.gz  95% |#############################  | ETA:  0:00:00  37.12 MB/s
packmol.tar.gz 100% |###############################| ETA:  0:00:00  34.66 MB/s
packmol.tar.gz 100% |###############################| Time: 0:00:00  31.35 MB/s
Extracting download
Package: packmol-1.0.0-0
source tree in: /Users/jenkins-slave/workspace/conda-omnia-release-osx-2/miniconda/conda-bld/work/packmol
Command failed: /bin/bash -x -e /Users/jenkins-slave/workspace/conda-omnia-release-osx-2/conda-recipes/packmol/build.sh
```

Is `packmol` needed for `omnia 1.0`?

**Comment by kyleabeauchamp** [comment]
Yes, might as well include it.

**Comment by jchodera** [comment]
Marked for omnia 1.0 milestone.

**Comment by jchodera** [comment]
This is fixed now by #75.  Thanks!
